src/components/Table/Table.css

```diff
@@ -3,15 +3,6 @@
 .container {
   min-width: 100%;
   clear: both;
-  @media only screen and (max-width: 960px) {
-    overflow-x: scroll;
-  }
-}
-
-.innerContainer {
-  @media only screen and (max-width: 960px) {
-    min-width: 1325px;
-  }
```

Comment:
I know this looked like the best way to do responsive content for every page. But this is not. This seriously sucks. Which is why I deleted it.

For an example on how to do responsive better, I have done it for Counts page. I will be following up on other pages in new PRs.